CHANGELOG.md

```diff
@@ -14,6 +14,25 @@
 - Added gRPC `/status` endpoint on all components (#817).
 - Added `--enable-otel` and `MIDEN_FAUCET_ENABLE_OTEL` flag to faucet (#834).
 
+## Unreleased
```

Comment:
I'm not sure this title is needed (it is superseded by the title on line 3 above).